docs/rules/no-chained-get.md

```diff
@@ -0,0 +1,28 @@
+# Disallow chain of `cy.get()` calls (`cypress/no-chained-get`)
+
+<!-- end auto-generated rule header -->
+This rule disallow the usage of chained `.get()` calls as `cy.get()` always starts its search from the cy.root element.
+
+## Rule Details
+
+Examples of **incorrect** code for this rule:
+
+```js
+cy.get('parent').get('child')
+```
+
+Examples of **correct** code for this rule:
+
+```js
+cy.get('parent')
+  .find('child')
+```
+
+```js
+cy.get('parent').within(() => {
+  cy.get('child')
+})
+```
+
+## Further Reading
+
```

Comment:
Possibly this should link to `[cy.get()](https://on.cypress.io/api/get)`

In the section https://docs.cypress.io/api/commands/get#Get-vs-Find it shows an example valid use of `cy.get('#comparison').get('div')`

```js
cy.get('#comparison')
  .get('div')
  // finds the div.test-title outside the #comparison
  // and the div.feature inside
  .should('have.class', 'test-title')
  .and('have.class', 'feature')
cy.get('#comparison')
  .find('div')
  // the search is limited to the tree at #comparison element
  // so it finds div.feature only
  .should('have.length', 1)
  .and('have.class', 'feature')
  ```

Reply:
@duranbe 

I suggest that you either remove the heading `## Further Reading` if there is no content for this section, or you link to the documentation page